package.json

```
@@ -1,7 +1,12 @@
 {
   "name": "@coveo/semantic-monorepo-tools",
   "version": "0.1.0",
-  "description": "",
+  "description": "A library of helper functions to do SemVer2 compliant releases from Conventional Commits in monorepos",
+  "readme": "README.md",
```

I've never seen the `readme` attribute in a `package.json` file but that doesn't mean it doesn't exist. ;)

So if it doesn't exist, you can change it to this one

```suggestion
  "homepage": "https://github.com/coveo/semantic-monorepo-tools#readme"
```

---

You're right @toofff , this was autosuggestion by IntelliSense, but this is probably used for VS Code Extensions or something like that.

homepage is the proper one.